plexus/CompositeScore.py

```diff
@@ -6,15 +6,22 @@
 from decimal import Decimal
 from abc import ABC, abstractmethod
 import tiktoken
+from pydantic import BaseModel
+import pandas as pd
 
 from plexus.processors.RelevantWindowsTranscriptFilter import RelevantWindowsTranscriptFilter
-from plexus.Score import Score
+from plexus.classifiers.Score import Score
```

Comment:
If we're going to move `Classifier` into `Score` in `plexus.classifiers` then it probably makes sense for that module to be called `plexus.scores`.  The whole reason I moved the abstract base class from `Classifier` to `Score` is that `Score` is more general, since they might not all be classifiers.